<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EaseEncrypt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(30, 30, 30)

}
QLabel{
color: rgb(200, 200, 200);
background-color:rgba(0, 0, 0, 0)
}

QPushButton {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(20, 115, 230);
	color: rgb(200, 200, 200);
border: 1px solid rgb(20, 115, 230);
}
QPushButton:pressed {
background-color:rgb(18, 100, 200);
color: rgb(200, 200, 200);
border: 1px solid rgb(18, 100, 200);
}
QPushButton:disabled {
background-color:rgb(20, 20, 20);
color: rgb(100, 100, 100);
border: 2px solid rgb(50, 50, 50);
border-radius: 10px;
}

QLineEdit{
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(20, 115, 230);
border-radius: 10px;
padding: 1px;
}

QTabWidget::pane {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgba(80, 80, 80, 255);
border-radius: 2px;
} 

QTabBar::tab {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgb(30, 30, 30);
border-radius: 2px;
padding: 8px;
} 

QTabBar::tab:selected { 
color: rgb(200, 200, 200);
background-color:rgb(18, 100, 200);
margin-bottom: -1px; 
}

QProgressBar {
background-color:rgb(30, 30, 30);
border: 1px solid rgb(80, 80, 80);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
background-color:rgb(20, 115, 230);
border: 1px solid rgb(20, 115, 230);
border-radius: 1px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_encrypt">
     <attribute name="title">
      <string>Encrypt</string>
     </attribute>
     <widget class="QLabel" name="lbl_e_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Output directory :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_e_input">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_e_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Input file(s) :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_e_output">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_encrypt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_decrypt">
     <attribute name="title">
      <string>Decrypt</string>
     </attribute>
     <widget class="QLabel" name="lbl_d_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Output directory :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="d_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_d_output">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="d_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_d_input">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_d_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Input file(s) :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_decrypt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="aniblock">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>405</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
background-color:rgb(30, 30, 30);
border: 2px solid rgba(80, 80, 80, 0);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.903727, stop:0 rgba(20, 115, 230, 255), stop:0.215909 rgba(20, 115, 230, 255), stop:0.528409 rgba(20, 115, 230, 226), stop:1 rgba(255, 255, 255, 0));
border: 2px solid rgba(20, 115, 230, 0);
border-radius: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>379</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_about">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_version">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>v0.1.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
